docs/blog/write-the-docs-2018-stats.rst

```diff
@@ -0,0 +1,75 @@
+.. post:: Jan 7, 2018
+   :tags: stats, year in review
+   :author: Eric Holscher
+
+Write the Docs 2018 Stats
+=========================
+
+2018 was another wonderful year for Write the Docs, it marks our 6th year of existence.
+
+This post contains stats that mark our progress,
+so that we can look back and see how we've changed as a community.
+
+This year I've removed a lot of the commentary except where it's useful for interpreting the numbers.
```

Comment:
You're assuming people have a) read previous posts and b) care to the same level you do :-p

Reply:
I'd prob just delete that sentence tbh

Reply:
Or work it into the next in a more standalone manner.

> I'm presenting the numbers without commentary, except where it's useful for interpreting the numbers, to facilitate comparison across the years.

Reply:
It looks like Eric is linking previous posts in the next sentence, I think it's fine to leave this. Or maybe flip them (so the links are first and then this note?)

Reply:
> Or work it into the next in a more standalone manner.
> 
> > I'm presenting the numbers without commentary, except where it's useful for interpreting the numbers, to facilitate comparison across the years.

I like this one the most. Explaining this year's stats seems more significant than comparing this report to previous reports. And if people wonder "why is there less commentary", this explains it too.